app/views/hyrax/base/_guts4form.html.erb

```diff
@@ -48,7 +48,7 @@
             </div>
       </div>   
     </div>
-
+  <% if @curation_concern.id  %>
```

Comment:
Lets do `@curation_concern.persisted?`. It's more explicit and avoids returning the extra data. Besides `id` can be set manually before persisting, so it's possible for this to be the wrong check.